server/app/org/scalaexercises/exercises/services/interpreters/interpreters.scala

```diff
@@ -153,18 +170,18 @@ trait TestInterpreters extends Interpreters[Id] with IdInstances {
   implicit val gitHubIdCaptureInstance = new GithubCapture[Id] {
     override def capture[A](a: ⇒ A): Id[A] = idMonad.pure(a)
   }
-
-  implicit val tailRecMTask = new RecursiveTailRecM[Id] {}
 }
 
 object FreeExtensions {
 
-  implicit class FreeOps[F[_], A](f: Free[F, A]) {
-    implicit val tailRecMTask = new cats.RecursiveTailRecM[scalaz.concurrent.Task] {}
-
-    def runFuture(implicit interpreter: F ~> Task, T: Transactor[Task], M: Monad[Task]): Future[Either[Throwable, A]] = {
+  implicit class FreeOps[F[_], A](f: FreeS[F, A]) {
+    def runFuture(
+      implicit
+      T: Transactor[Task],
+      M: MonadError[Task, Throwable]
```

Comment:
What happens if you add an implicit `F ~> Task` interpreter here?
With `freestyle.implicits._` in the files where you call `.runFuture`.